lib/debug/debugger_ui.dart

```diff
@@ -120,6 +121,9 @@ class DebuggerView extends DockedView {
     subs.add(connection.onPaused.listen(_handleIsolatePaused));
     subs.add(connection.onResumed.listen(_handleIsolateResumed));
     subs.add(connection.isolates.onRemoved.listen(_handleIsolateTerminated));
+
+    // Optional events
+    subs.add(connection.onLibrariesChanged?.listen(_handleLibrariesChanged));
```

Comment:
Ah, this will add a `null` subscription sometimes. You may instead want to re-write to use an if statement.

Reply:
Done.